ControllerProject/Service/UserService.cs

```diff
@@ -44,6 +79,25 @@ public User GetUser(string username)
             return user;
         }
 
+        public AddUser GetAddUser(int id)
+        {
+            var loginInfoId = FindLoginInfoesIdByUserId(id);
+            var loginInfo = (from l in _context.LoginInfoes
+                        where l.Id == loginInfoId
+                        select l).FirstOrDefault<LoginInfo>();
+            var user = GetUser(FindUsernameById(id));
+            AddUser addUser = new AddUser(
+                loginInfo.Username,
+                loginInfo.Password,
+                loginInfo.Email,
+                user.Firstname,
+                user.Lastname,
+                user.AccessRights
+            );
+
+            return addUser;
```

Comment:
Galbūt labiau būtų tikęs metodas, kuris grąžina user tada nereikėtų dar susikurti papildomo struct. Čia taip pat gali iš loginInfo gauti user, kuriam priklauso nereikės šito FindUsernameById() metodo.

Reply:
Same

Reply:
Be to, FindUsernameById dar naudojamas ManageUsersController DeleteUser metode